Overlays/Posix/Overlays/Android/Libraries/WNScripting/CMakeLists.txt

```diff
@@ -39,12 +39,9 @@ if (WN_OVERLAY_IS_ENABLED AND NOT WN_DISABLE_CXX)
     "-DLLVM_EXPORT_SYMBOLS_FOR_PLUGINS=OFF")
   append_to_overlay_var(WN_LLVM_EXTRA_FLAGS
     "-DCMAKE_CXX_FLAGS=-D__ANDROID_NDK__")
-  find_program(PYTHON python)
-
-  append_to_overlay_var(WN_LLVM_EXTRA_FLAGS
```

Comment:
This didn't seem to get used on android. Cmake reports that `PYTHON_EXECUTABLE` variable is not used.